### DISCUSSION TITLE
How to move settings folder?

### DISCUSSION THREAD

**Discussion by pstein** [audacious-media-player/audacious#1481] [General]
Currently all settings are stored (in Windows 10) in

C:\Users\ps\AppData\Local\audacious\

How can I move the settings folder to another directory (e.g. sub directory of Audacious)?

Either with an absolute path or (preferred) with an relative path (relative to installation folder of Audacious)

**Comment by radioactiveman** [comment]
I assume this is not intended. You may use a symbolic link (aka symlink) for the directory. Not sure though if they work the same on Windows.

**Comment by PaduaPlay** [comment]
Defaulting the settings folder to the C:\Users\ *** \Local\audacious\ directory makes this not portable.

Is there a command line or .ini file that can be added to have the settings directory in the program directory itself?

Having a way to do it within the program is simpler than a symlink.

Audacious 4.4.1 on Windows 11.

**Reply by radioactiveman** [reply]
If you want Audacious to be portable, you may use Audacious from https://portableapps.com/apps/music_video/audacious-portable.
